tests/cpu/Config_tests.cpp

```diff
@@ -3749,6 +3749,271 @@ OCIO_ADD_TEST(Config, display_view_order)
     OCIO_CHECK_EQUAL(std::string(config->getView("sRGB_B", 1)), "View_1");
 }
 
```

Comment:
Please move this test to tests/cpu/Display_tests.cpp.

And please add a test of clearSharedViews to the "basic" test in that module.

Reply:
Sorry, my instructions were not clear.  Please move the new test "compare_virtual_displays" to Display_tests.cpp and restore the old test "virtual_display_exceptions" to Config_tests.cpp.